rllib/algorithms/simple_q/simple_q_torch_policy.py

```diff
@@ -60,40 +60,45 @@ def make_model(self) -> ModelV2:
         return model
 
     @override(TorchPolicyV2)
-    def compute_actions(
+    def compute_actions(self, *args, **kwargs):
```

Comment:
Since simple q needs it's own compute actions functions, do the same thing as in qmix here.
-> We ourselves only ever use compute_actions_from_input_dict.